### PR TITLE
More details and docs

### PR DESCRIPTION
**Pull request by hollyallen**
Added detail about creating the ATO folder, added more documents to create. Trying to make this list more self-service and complete.

/cc @afeld

### CODE REVIEW

File: _pages/ato/checklist.md [hunk 1]
```diff
@@ -31,19 +31,22 @@ Many of these tasks can be done in parallel. This is just a suggestion of priori
...
+        * [ ] Add the issue-free scan report or [documentation about false positives](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/#caveats) to the ATO folder.
+5. [ ] [Update relevant documentation](https://pages.18f.gov/before-you-ship/ato/tips/), primarily the README
+6. [ ] If this will be a new ATO, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to any currently-running sites
+7. [ ] Add a [System Security Plan yml](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
+8. [ ] Add a System Security Plan Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
+9. [ ] [Set up Compliance Masonry documentation](https://github.com/18F/cg-compliance#starting-ato-documentation-for-cloudgov-applications)
+10. [ ] [Implement the controls](https://pages.18f.gov/before-you-ship/ato/walkthrough/#step-3--implement-the-controls)
+11. [ ] Add a Rules of Engagement Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
+12. [ ] Add a Project Plan Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
```
Comment (afeld) [inline _pages/ato/checklist.md]: From https://daringfireball.net/projects/markdown/syntax#list:

> the actual numbers you use to mark the list have no effect on the HTML output Markdown produces

Therefore, it's preferable to start them all with the same number, so you don't need to renumber manually, and can avoid big confusing diffs like this one 😉 Mind changing them back?

P.S. GitHub Markdown doesn't show the numbers when it's a checkbox list, so this is all a bit moot.

Comment (hollyallen) [inline _pages/ato/checklist.md]: Thanks! should we just remove the numbers in that case?

Comment (afeld) [inline _pages/ato/checklist.md]: Welllll, if we want to be pedantic about it, they _are_ ordered...it's just a limitation of GitHub that they aren't shown that way 😁 If you don't want to change them again, feel free to merge.

**Comment by hollyallen** [comment]
You approved the changes, but do you merge the PR or do I?

**Comment by afeld** [comment]
Sorry, got pushed back in my inbox!

**Pull request merged**